skglm/estimators.py

```diff
@@ -423,7 +422,11 @@ def fit(self, X, y):
         self :
             Fitted estimator.
         """
-        return _glm_fit(X, y, self, Quadratic(), L1(self.alpha))
+        solver = AcceleratedCD(
```

Comment:
We want to be more flexible than that: pass `solver` to Lasso and call self.solver here

Reply:
on second thoughts we want to be able to instantiate estimator with providing the solver. 
A default choice could be solver=None of solver='cd' and instantiate a solver at fit time.
solver=instance of BaseSolver should also be supported